README.md

```diff
@@ -58,6 +58,14 @@ contact_phone: "(212) 555-1212"
 contact_email: "format@shunn.net"
 ```
 
+Optional properties for the frontpage:
```

Comment:
I don't see where these optional fields are used. Did you mean to implement them in the template(s)? If this is partial work, please take them out of the README and open an issue. We can incorporate them in a separate PR.

Reply:
They're within the template.docx files
![image](https://user-images.githubusercontent.com/41603229/152662830-63cb4270-a04b-456a-a2f7-509637714d47.png)

Oh, looks like there's a `date` too

**Result:**
![image](https://user-images.githubusercontent.com/41603229/152662832-9a85a483-9012-4941-9321-cadc3ac9c720.png)

Reply:
Oh! Yes, you're right. My mistake. I was thinking you were trying to add more custom metadata like I had to do for short_title, contact_name, etc. Yes, `subtitle`, `date`, and `abstract` are valid options and should be documented like you did. Thank you!